File: LeetcodeQuestions/running_sum_of_1d_array-1480.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        sum = 0
        sums = [0] * len(nums)
        for i in range(len(nums)):
            sum += nums[i]
            sums[i] = sum
        return sums

File: LeetcodeQuestions/runningSumOf1DArray-1480.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */

var runningSum = function(nums) {
    let sum = 0;
    let sums = new Array(nums.length);
    for (let i = 0; i < nums.length; i++) {
        sums[i] = sum += nums[i];
    }
    return sums
};